### PR TITLE
(paint.net) embedded + modernized

### LINKED ISSUE #861: (paint.net) Embed the package
> Redistribution of the "Classic" release of Paint.NET is permitted so long as it is free-of-charge

### PR DESCRIPTION
**Pull request by majkinetor**
closes #861

### CODE REVIEW

File: automatic/paint.net/legal/VERIFICATION.txt [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+   to download the installer.
+
+2. You can use one of the following methods to obtain the SHA256 checksum:
+   - Use powershell function 'Get-FileHash'
+   - Use Chocolatey utility 'checksum.exe'
+
+   checksum32: F254361CAB183D637D4915B6FA321254132158D02491AD5243357868C0B07785
+
+File 'license.txt' is obtained from:
+   hhttps://www.getpaint.net/license.html
```
Comment (gep13) [inline automatic/paint.net/legal/VERIFICATION.txt]: @majkinetor slight typo here? `hhttps` should be `https`

Comment (majkinetor) [reply automatic/paint.net/legal/VERIFICATION.txt]: Thx. fixed

**Comment by AdmiringWorm** [comment]
@majkinetor seems like both me and @gep13 missed it, but the legal folder isn't included at all.
Mind adding it?

**Comment by majkinetor** [comment]
Sure.

**Pull request merged**